go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/googlecloudplatform/gcs-fuse-csi-driver
 
-go 1.22.0
-
-toolchain go1.22.2
```

Comment:
Deleting toolchain looks a little suspicious. Not sure how bumping to 1.23 removes the need for that

Reply:
Actually, looks like that is what we do in main branch too. https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/blob/bd50eca4b7a0ac734de69fa3429a128cd4dfa1d3/go.mod#L3 Disregard